src/usr/local/www/services_dhcp.php

```diff
@@ -1402,6 +1420,15 @@ function build_pooltable() {
 						<td ondblclick="document.location='services_dhcp_edit.php?if=<?=htmlspecialchars($if)?>&amp;id=<?=$i?>';">
 							<?=htmlspecialchars($mapent['mac'])?>
 						</td>
+<?php
+			if ($got_cid) {
```

Comment:
Please use if (...): / else: / endif; style in cases where HTML content is printed inside the conditional